### DISCUSSION TITLE
[Help]: packages/embla-carousel-auto-scroll is not available on NPM.

### DISCUSSION THREAD

**Discussion by RenlyLtd** [davidjerleke/embla-carousel#721] [Help] [answered]
### Summary

When i try to import from NPM i get an error saying that package is not available.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [X] embla-carousel-vue
- [X] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
@RenlyLtd at this point this is expected because of things I didn't know and couldn't control. Please [read this](https://github.com/davidjerleke/embla-carousel/issues/114#issuecomment-1911757602).

Best,
David
